test/webvr-polyfill.test.js

```diff
@@ -0,0 +1,150 @@
+import WebVRPolyfill from '../';
+import { setupWindow, setIOSUA, setAndroidUA, addNativeNoDisplay, addNativeWithDisplay } from './utils';
+import chai from 'chai';
+const { expect } = chai;
+
+const DEFAULT = {
+  DPDB_URL: null,
+};
+
+describe('WebVRPolyfill', function() {
+  beforeEach(() => setupWindow(global));
+  afterEach(() => delete global.window);
+
+  it('populates globals if enabled', () => {
+    new WebVRPolyfill(DEFAULT);
+    expect(global.navigator.getVRDisplays).to.exist;
+    expect(global.window.VRDisplay).to.exist;
+    expect(global.window.VRFrameData).to.exist;
+  });
+
+  it('should be enabled if API does not exist', () => {
+    expect(global.window.getVRDisplays).to.not.exist;
+    const polyfill = new WebVRPolyfill(DEFAULT);
+    expect(global.navigator.getVRDisplays).to.exist;
+    expect(polyfill.enabled).to.be.true;
+  });
+
+  it('should be disabled on desktop if API exists and PROVIDE_MOBILE_VRDISPLAY=false', () => {
+    addNativeWithDisplay(global.window, '1.1');
+    expect(global.navigator.getVRDisplays).to.exist;
+    const polyfill = new WebVRPolyfill(Object.assign({
+      PROVIDE_MOBILE_VRDISPLAY: false,
+    }, DEFAULT));
+    expect(global.navigator.getVRDisplays).to.exist;
+    expect(polyfill.enabled).to.be.false;
+  });
+
+  it('should be disabled on mobile if API exists and PROVIDE_MOBILE_VRDISPLAY=false', () => {
+    setIOSUA(global);
+    addNativeWithDisplay(global.window, '1.1');
+    expect(global.navigator.getVRDisplays).to.exist;
+    const polyfill = new WebVRPolyfill(Object.assign({
+      PROVIDE_MOBILE_VRDISPLAY: false,
+    }, DEFAULT));
+    expect(global.navigator.getVRDisplays).to.exist;
+    expect(polyfill.enabled).to.be.false;
+  });
+
+  it('should be enabled on mobile if API exists and PROVIDE_MOBILE_VRDISPLAY=true', () => {
+    setIOSUA(global);
+    addNativeWithDisplay(global.window, '1.1');
+    expect(global.navigator.getVRDisplays).to.exist;
+    const polyfill = new WebVRPolyfill(Object.assign({
+      PROVIDE_MOBILE_VRDISPLAY: true,
+    }, DEFAULT));
+    expect(global.navigator.getVRDisplays).to.exist;
+    expect(polyfill.enabled).to.be.true;
+  });
+
+  describe('hasNative', () => {
+    it('false when no native support found', () => {

```

Comment:
standard it string would be 'should be false when no native support found'. Same for below (add 'should be')

Reply:
:+1: